src/applications/ezr/utils/helpers/form-config.js

```diff
@@ -47,6 +48,43 @@ export function includeTeraInformation(formData) {
   return formData.hasTeraResponse;
 }
 
+export function veteranBornBetween(formData, date1, date2) {
+  return moment(formData?.veteranDateOfBirth).isBetween(
```

Comment:
Did you look into using any of these `date-fns` functions? I know moment.js has some performance issues and it looks like we have some linters to steer away from it. But clearly we still use it all over the place 🤷

Reply:
https://blog.openreplay.com/what-is-better--datefns-or-moment/ For reference

Reply:
Also found this helpful although a little old https://github.com/you-dont-need/You-Dont-Need-Momentjs

Reply:
I talked with Matt L. and he agreed that we should move away from `moment` with our apps. With that said, I removed `moment` code and replaced it with `date-fns` code!